tests/test_slack.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from karmabot.slack import get_available_username, get_slack_id, get_user_id
+
+
+@pytest.mark.parametrize(
+    "test_profile, expected",
+    [
+        ({"display_name_normalized": "pybob", "real_name_normalized": "Bob"}, "pybob"),
+        (
+            {
+                "display_name_normalized": None,
+                "real_name_normalized": "Julian Sequeira",
+            },
+            "Julian Sequeira",
+        ),
+    ],
+)
+def test_get_available_username(test_profile, expected):
+    assert get_available_username(test_profile) == expected
+
+
+def test_get_available_username_wrong_format():
+    with pytest.raises(ValueError) as exc:
```

Comment:
You can use the `match` keyword: https://twitter.com/pybites/status/1410883387984859138

Reply:
nice! done!

Reply:
Recently learned this, nice feature, `pytest` keeps amazing me.